tendrl/commons/jobs/__init__.py

```diff
@@ -24,7 +24,7 @@ def _process_job(self, raw_job, job_key):
             # Generate a request ID for tracking this job
             # further by tendrl-api
             req_id = str(uuid.uuid4())
-            if tendrl_ns.type == "node":
+            if tendrl_ns.type == "node" or tendrl_ns.type == "monitoring":
```

Comment:
how many types does performance and alerts repo define? this is something the tendrl-api decides, please clarify @anmolbabu 

cc @anivargi